src/game-board/game-board.html

```diff
@@ -105,49 +122,40 @@
       *
       * @param {String} symbol - utf-8 representation of card string.
       */
-      _convertSuit(symbol){
+      _convertSuit(symbol) {
         switch(symbol){
           case "♥":
             return 'heart';
-          break;
           case "♠":
-              return 'spade';
-          break;
```

Comment:
I would like to know the reasoning behind removing the breaks, is it because they are simply unneeded since the cases are so different from each other?